snmp/datadog_checks/snmp/data/default_profiles/hp-h3c-switch.yaml

```diff
@@ -0,0 +1,68 @@
+extends:
+  - _generic-if.yaml
+  - _hp.yaml
```

Comment:
`_hp.yaml` defines the vendor and `_hp-base.yaml` defines the metadata
I didn't want to use `_hp-base.yaml` with this profile since they use different MIBs